debian/rules

```diff
@@ -54,6 +54,7 @@ DEPENDS += ansible, \
 	   debsums, \
 	   dmidecode, \
 	   net-tools, \
+	   ntp, \
```

Comment:
Does this address [TOOL-12422](https://jira.delphix.com/browse/TOOL-12422)?

Reply:
No, it doesn't, it's just a backport from master. It does address the current failure to build the internal-dcenter variant though.